Whiskey/Functions/Invoke-WhiskeyNUnit2Task.ps1

```diff
@@ -127,6 +127,12 @@ function Invoke-WhiskeyNUnit2Task
     {
         Stop-WhiskeyTask -TaskContext $TaskContext -Message ('{0} {1} was installed, but couldn''t find nunit-console.exe at ''{2}''.' -f $package,$version,$nunitConsolePath)
     }
+

```

Comment:
Installation of all the tools needs to happen before validating the paths property (about line 70). On a fresh computer, the paths for the task won't exist and initialization will fail. Add a test that ensures this.

Reply:
Done